test/run_tests.py

```diff
@@ -10,26 +10,43 @@
 
 Usage:
 
-  run_tests.py interpreter browser [options]
+  run_tests.py browser [options]
 
 Arguments:
 
-  interpreter:  Any Python interpreter supported by the library (e.g. `python`,
-                `jython`, `c:\\Python27\\python.exe`)
-  browser:      Any browser supported by the library (e.g. `chrome`, `firefox`)
-  options:      Additional command line options passed to Robot Framework
+  browser:        Any browser supported by the library (e.g. `chrome`or
+                  `firefox`)
+  --interpreter:  Any Python interpreter supported by the library (e.g.
+                  `python`, `jython` or `c:\\Python27\\python.exe`). By
+                  default set to `python`.
+  --suite:        Selects the test suites by name.
+  --scusername:   Username to access Sauce Labs to order browsers when
+                  running test from local computer.
+  --sckey:        Access key for Sauce Labs account.
+
+When running test by using browser from Sauce labs, it is required that
+the Sauce Connect is used. The Sauce Connect allows the browser from Sauce Labs
+reach the acceptance test webserver. The acceptance test uses tunnel with
+name `localtunnel` and therefore when establishing  the Sauce Connect tunnel
+use the following command:
+`bin/sc -u YOUR_USERNAME -k YOUR_ACCESS_KEY -i localtunnel`
+
+More details and to downlaod Sauce Connect visit:
+https://wiki.saucelabs.com/display/DOCS/High+Availability+Sauce+Connect+Setup
 
 Examples:
 
   run_tests.py python chrome
-  run_tests.py jython c:\\Python35\\python.exe --test "Click element"
+  run_tests.py jython firefox --suite list
```

Comment:
There was a change in the argument passing (mentioned on doc), but the actual code does not look like it has been changed.
Example should be:
`run_tests.py --interpreter jython firefox --suite list`

There are more fixes like on some typos. I'll stop the review here, but will test this file run_tests.py for better review.